### PR TITLE
[NEW] New button for network outage

### PR DESCRIPTION
**Pull request by amolghode1981**
Description: Added a new icon for network disconnection. This icon will show up whenever there is network disconnection.

## Proposed changes (including videos or screenshots)

When network outage happens it should be conveyed to the user with special icon. This icon should not be clickable.
Network outage handling is handled in https://app.clickup.com/t/245c0d8 task.

## Issue(s)
https://app.clickup.com/t/29wgna9
https://app.clickup.com/t/245c0d8

## Steps to test or reproduce
This can be reproduced by turning off the network or stopping asterisk.

### CODE REVIEW

File: apps/meteor/package.json [hunk 1]
```diff
@@ -195,12 +195,12 @@
 		"@rocket.chat/emitter": "~0.31.11",
 		"@rocket.chat/forked-matrix-appservice-bridge": "^4.0.1",
 		"@rocket.chat/forked-matrix-bot-sdk": "^0.6.0-beta.2",
-		"@rocket.chat/fuselage": "~0.31.11",
+		"@rocket.chat/fuselage": "next",
```
Comment (amolghode1981) [inline apps/meteor/package.json:198]: @MartinSchoeler latest didnt work. As pointed by @murtaza98 , the tagging for icons and fuselage repo was not latest but "next". So this worked.

Though I have a question. I am sure we can not merge this file with _**"@rocket.chat/fuselage": "next",**_
If so, what version should we put here while merging the code with develop?

Comment (murtaza98) [inline apps/meteor/package.json:198]:
```suggestion
		"@rocket.chat/fuselage": "~0.32.0-dev.33",
```

File: apps/meteor/package.json [hunk 2]
```diff
@@ -195,12 +195,12 @@
...
 		"@rocket.chat/fuselage-hooks": "~0.31.11",
 		"@rocket.chat/fuselage-polyfills": "~0.31.11",
 		"@rocket.chat/fuselage-tokens": "~0.31.11",
 		"@rocket.chat/fuselage-ui-kit": "~0.31.11",
-		"@rocket.chat/icons": "~0.31.11",
+		"@rocket.chat/icons": "next",
```
Comment (murtaza98) [inline apps/meteor/package.json:203]:
```suggestion
		"@rocket.chat/icons": "~0.32.0-dev.15",
```

File: apps/meteor/package.json (hunk 1 above)
Comment (amolghode1981) [reply apps/meteor/package.json]: Fixed.

File: apps/meteor/package.json (hunk 2 above)
Comment (amolghode1981) [reply apps/meteor/package.json]: Fixed.

File: apps/meteor/client/sidebar/sections/components/OmnichannelCallToggleReady.tsx [hunk 3]
```diff
@@ -31,10 +31,24 @@ export const OmnichannelCallToggleReady = (): ReactElement => {
...
 	};
 
+	const getIcon = (): 'phone-issue' | 'phone' | 'phone-disabled' => {
+		if (networkStatus === 'offline') {
+			return 'phone-issue';
+		}
+		return registered ? 'phone' : 'phone-disabled';
+	};
+
+	const getColor = (): string | undefined => {
```
Comment (MartinSchoeler) [inline apps/meteor/client/sidebar/sections/components/OmnichannelCallToggleReady.tsx:41]:
```suggestion
	const getColor = (): 'warning' | 'success' | undefined => {
```

**Review by MartinSchoeler — APPROVED** [review]

**Review by MartinSchoeler — APPROVED** [review]

**Pull request merged**